### PR TITLE
Feat: #118 : Redesigned the Navbar component utilizing MUI

### LINKED ISSUE #118: feat: Redesign the Navbar component
### **Is your feature request related to a problem? Please describe.**
Redesign the `navbar` component.

### **Describe the solution you'd like**
Redesign the navbar to ensure it's visually appealing, consistent across pages, and flexible in its layout. The specifications are:
- Utilize `Material-UI (mui)` and `CSS` for styling.
- The Navbar should be Sticky.
- **There should be a single navbar component adaptable to the home screen and the other screens**.
- Use the[ MUI App Bar Component](https://mui.com/material-ui/react-app-bar/).
- The functionality should be same as the existing Navbar.

### **Additional context**

- Screenshots of the desired navbar design:
image
- The exact design can be found in this [figma file](https://www.figma.com/design/vDVEoB5YY32EUoeFWtXaCA/Lupo-Skill?node-id=174-786&t=upBP1TkxavtzxW1i-1).

- This screen recording shows how get exact colors and values from a figma design.

https://github.com/Tanay-ErrorCode/lupo-skill/assets/116624667/c1fe8e14-50af-46b6-b473-5dbeecbe3748

### PR DESCRIPTION
**Pull request by Utsavladia**
**What kind of change does this PR introduce?**

Redesigned the Navbar component as provided in the Figma

 utilizing Material-UI and CSS for styling 
**Issue Number:**

Fixes #118 
**Snapshots/Videos:**

Screenshot 2024-05-27 000052
Screenshot 2024-05-26 234016

**Other information**

I made it Responsive for best user experience..
Thank you for giving me this opportunity.

### CODE REVIEW

**Comment by Doraemon012** [comment]
@Utsavladia 

  - It was mentioned in the issue that the navbar has different design for the home screen, and the other pages.

  - Navbar for **Home Screen**:
  
  image
  
 -  Navbar for **other pages**:
  
  image
  
  - Also, the link to figma designs was provided so that the designs can be viewed there.
  - The the navbar should be adjustable (through props) for the two designs (for home page and for other pages)
  - Please use the colors for buttons, from `theme.tsx`
  - In addition to the colors in `theme.tsx` use these colors for the `Home Page Navbar`
  
    ```css
    home-nav-background: 'rgba(255, 255, 255, 0.17)', // ffffff with 17% opacity
    home-nav-inactiveButton: 'rgba(252, 252, 252, 0.20)', // fcfcfc with 20% opacity
    ```

File: src/components/Cards/CreateEvent/CreateEvent.tsx [hunk 1]
```diff
@@ -287,11 +287,36 @@ const CreateEvent = ({ onNavLinkClick, props }: any) => {
...
+            },
+          }}
+        >
           Create Event
-        </Button>
+        </MenuItem>
       ) : (
-        <Nav.Link onClick={handleShow}>Create Event</Nav.Link>
+        <Button
+          variant="contained"
```
Comment (Doraemon012) [inline src/components/Cards/CreateEvent/CreateEvent.tsx:306]: Why have you changed these buttons ?
Please revert this.

File: src/components/Cards/CreateEvent/CreateEvent.tsx [hunk 2]
```diff
@@ -373,10 +398,14 @@ const CreateEvent = ({ onNavLinkClick, props }: any) => {
           </Form>
         </Modal.Body>
         <Modal.Footer className="modal_back">
-          <Button variant="secondary" onClick={handleClose}>
```
Comment (Doraemon012) [inline src/components/Cards/CreateEvent/CreateEvent.tsx:376]: Why have you changed these buttons ?
Please revert this.

**Review by Doraemon012 — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by Utsavladia** [comment]
> Please make the requested changes.

Thank you for the feedback, I will try to fix it.

**Pull request closed without merging**